### DISCUSSION TITLE
Tiptop editor default outline

### DISCUSSION THREAD

**Discussion by Erol-Mehmed-SourceCode** [ueberdosis/tiptap#5951] [General Questions] [answered]
I have a red outline by default, which I think comes from a pseudoelement, because when I remove position: relative on .MuiTiptap-FieldContainer-root element the outline dissapears. The problem is I cannot set hover and focus to this element after I remove position: relative.

I did not manage to find solution online. Can you help, I want to style hover and focus of this element?

**Comment by nperez0111** [comment, marked as answer]
Styling is not really a tiptap concern. Also you seem to be using [mui-tiptap](https://github.com/sjdemartini/mui-tiptap) based on that classname so even more not our problem
